docs/reference/api.md

```diff
@@ -0,0 +1,77 @@
+# API
+
+Pebble exposes an API to allow remote clients to interact with the daemon. The API can start and stop services, add configuration layers to the plan, and so on.
+
+## Accessing the API
+
+The API uses HTTP over a Unix socket, with access to the API controlled by user ID. If `pebble run` is started with the `--http <address>` option, Pebble exposes a limited set of open-access API endpoints using the given TCP address. See {ref}`api-access-levels`.
+
+See below for some examples of how to use the API. For more examples, see "How to use the Pebble API". <!-- [David] Link to the how-to guide -->
+
+There's a Go client for the API. See the [Go client documentation](https://pkg.go.dev/github.com/canonical/pebble/client) and the examples below.
+
+We try to never change the underlying HTTP API in a backwards-incompatible way. However, in rare cases we may change the Go client in a backwards-incompatible way.
+
+There's also a Python client for the API. The Python client is part of the `ops` library used by Juju charms. See the [Ops documentation](https://ops.readthedocs.io/en/latest/).
+
+### Go client examples
+
+TODO: go client code snippets.
+
+### cURL examples
+
+TODO: curl examples.
+
+## API access levels
```

Comment:
For now, can we just drop this section (including "Identities") and have this content at `/explanation/api-and-clients`. I think we should merge the two, but not needed in this PR.